x/ccv/provider/keeper/grpc_query.go

```diff
@@ -139,6 +145,7 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 		MinStake:                powerShapingParameters.MinStake,
 		ConsumerId:              consumerId,
 		AllowlistedRewardDenoms: &types.AllowlistedRewardDenoms{Denoms: allowlistedRewardDenoms},
+		Prioritylist:            strPrioritylist, //strPrioritylist, Should I retrieve from somewhere => where do I retrieve it from? Basically where does priority list come from and where is it stored
```

Comment:
You got it right! just make sure to remove the comment :)

Reply:
Okay, thank you!